woocommerce/woocommerce-gateway-payfast
# fix/167: Add WooPayments compatibility

### All Submissions:



---

### Changes proposed in this Pull Request:


Closes #167 .

### Steps to test the changes in this Pull Request:



1. Set USD as the store currency.
2. Add `ZAR` to WooPayments multi-currency.
3. Ensure WooPayments and Payfast are both enabled.
4. Visit the shop page and set the currency to `ZAR`, add a product to cart and visit checkout page.
5. Observe Payfast gateway is not in the list.
6. Switch to fix branch and repeat (4).
7. Observe Payfast is in the list.

### Changelog entry


> Fix - Payfast gateway not visible on Checkout when `ZAR` currency is set via WooPayments multi-currency feature.
